source/agora/test/Flash.d

```diff
@@ -593,19 +593,17 @@ private class FlashListener : TestFlashListenerAPI
 
     public Amount getEstimatedTxFee ()
     {
-        return Amount(1);
+        return flashTestConf().consensus.min_fee;
     }
 }
 
-private TestConf flashTestConf ()
+private TestConf flashTestConf () @safe
 {
     import agora.node.Config;
 
     TestConf conf;
     conf.consensus.quorum_threshold = 100;
-    // TODO: remove this line when fees are handled
-    conf.consensus.min_fee = Amount(0);
-    conf.node.min_fee_pct = 0;
+    conf.consensus.min_fee = Amount(1);
```

Comment:
Much nicer, but it still looks like a hack though. Lowering the minimum fee is not something we should be doing if we can avoid it. Can we avoid it ?